librz/core/core.c

```diff
@@ -2663,7 +2675,7 @@ RZ_API void rz_core_fini(RzCore *c) {
 	sdb_free(c->sdb);
 	rz_parse_free(c->parser);
 	free(c->times);
-	rz_core_seek_free(c);
```

Comment:
??  why?

Reply:
That seems to be an unfortunate typo. I will fix it.